guidelines.json

```diff
@@ -3409,7 +3409,7 @@
 						{
 							"title": "Search engines",
 							"testable": "[Machine-testable](https://w3c.github.io/sustainableweb-wsg/star.html#WD09-2)",
-							"description": "Search engines are not obstructed, while ill-intentioned robots and scripts are blocked.",
+							"description": "Permit unobstructed access to search engines while blocking ill-intentioned robots and scripts.",
```

Comment:
Is this criterion sufficiently relevant to sustainability? And if it is, is it scoped right? 

I see what you're getting at, but would suggest to drop it for two reasons: 

- 'Ill-intentioned' seems impossible to measure
- Outcomes matter more than intentions; all crawlers had some good intention at some point, so they would pass, but despite that, especially the AI training ones have huge outcomes on sustainability (see sysadmins quoted in [this Ars Technica piece](https://arstechnica.com/ai/2025/03/devs-say-ai-crawlers-dominate-traffic-forcing-blocks-on-entire-countries/)).